core/src/main/java/io/questdb/griffin/engine/groupby/AbstractNoRecordSampleByCursor.java

```diff
@@ -47,6 +48,7 @@ public abstract class AbstractNoRecordSampleByCursor extends AbstractSampleByCur
     protected RecordCursor baseCursor;
     protected Record baseRecord;
     protected SqlExecutionCircuitBreaker circuitBreaker;
+    protected boolean endFill;
```

Comment:
Can we move this field into the `SampleByFillValueNotKeyedRecordCursor` class where it belongs logically?